### PR TITLE
BookStore: Milestone two

### PR DESCRIPTION
**Pull request by Emmyn5600**
### In this PR, we implemented the following features:
* Prepared or Created the directories structure such as `components, actions, reducers and containers`.
* Combined the reducers in the index.js file inside the reducers folder.
* Implemented the App component.
* Implemented the BooksList component.
* Implemented the Book component.
* Implemented the BooksForm component.
* Defined the Redux actions and reducers.
* Initialized the Redux store.

### CODE REVIEW

File: src/containers/BooksForm.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <form>
+        <input type="text" placeholder="title" />
+        <select name="category" id="category">
+          {categories.map((categ) => (
+            <option value={categ} key={categ}>
+              {categ}
+            </option>
+          ))}
+        </select>
+      </form>
```
Comment (oloomoses) [inline src/containers/BooksForm.jsx]: - [x] This for is missing a submit button, kindly add it to enable a user to submit the book category as per the  [requirements from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Form/RenderForm.png)

File: src/reducers/books.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { CREATE_BOOK } from '../actions';
+
+const books = (books = [], action) => {
+  switch (action.type) {
+    case CREATE_BOOK:
+      return [...books, action.payload];
+    default:
+      return books;
+  }
+};
```
Comment (oloomoses) [inline src/reducers/books.js]: - [x] Kindly implement the `REMOVE_BOOk` to enable a user to remove a book as per the project  [requirements from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/ReducxReducers/CreateAndRemoveReducers.png)

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## Required Changes ♻️
By the end of this milestone Kindly ensure that a user is able to:
- [ ] Create a book by clicking the submit button
- [ ] Remove a book by clicking the remove button

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Slack: Oloo Moses_

**Comment by barackm** [comment]
Hi code Reviewer,  the two requirements you mentioned are supposed to be implemented in the next milestone (milestone 3)
➡️  Making that form functional is the goal of the next Milestone.

**Review by rukundoeric — APPROVED** [review]
Hi @barackm  and @Emmyn5600 ,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**